tests/bluetooth/common/testlib/src/adv.c

```diff
@@ -35,7 +35,7 @@ static void connected_cb(struct bt_le_ext_adv *adv, struct bt_le_ext_adv_connect
 	k_mutex_unlock(&g_ctx_lock);
 }
 
-int bt_testlib_adv_conn(struct bt_conn **conn, int id, uint32_t adv_options)
+int bt_testlib_adv_conn(struct bt_conn **conn, int id)
```

Comment:
Doesn't it make sense to keep the options here? Since this is a library, tests may still want to set other options.

Reply:
@alwa-nordic what do you think, IMO it's not useful to have options here as it's a very specific function used to create a connection, not just start an advertiser